spaceship/models/mission.py

```diff
@@ -22,3 +22,24 @@ class Mission(BaseModel):
   @hybrid_property
   def is_active(self):
     return self.deleted_at == None
+
+  @hybrid_property
```

Comment:
i think you want this (and others here) to just be an `@property` -- `@hybrid_property` is for things that can be used to build DB queries. http://docs.peewee-orm.com/en/latest/peewee/playhouse.html#hybrid-attributes

Reply:
ok! sounds like we can always convert it back to hybrid if we need those queries.